http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/ClusterHelper.scala

```diff
@@ -619,6 +627,34 @@ class ClusterHelper(
     }
   }
 
+  private def getMasterInstanceShutdownScript(cluster: Cluster): IO[Map[String, String]] = {
+    val googleKey = "shutdown-script" // required; see https://cloud.google.com/compute/docs/shutdownscript
+
+    // These things need to be provided to ClusterInitValues, but aren't actually needed for the shutdown script
+    val dummyInitBucket = GcsBucketName("")
+    val dummyStagingBucket = GcsBucketName("")
```

Comment:
Would it be better to not overload `ClusterInitValues` for this purpose? If we must, should the `dummy` fields be optional?

Reply:
Good point.. I like having `ClusterInitValues` used for init, start, and shutdown scripts because it means each script has access to all templated variables. Totally agree about making the dummy fields optional.. will try and fix.

Reply:
Did this in latest commit: https://github.com/DataBiosphere/leonardo/pull/1170/commits/45b7f1231b093f8cd42377f24b9dfff28c8e0e51